internal/blockchain/ethereum/abi_definitions.go

```diff
@@ -96,3 +96,16 @@ var batchPinEventABI = ABIElementMarshaling{
 		},
 	},
 }
+
+var networkVersionMethodABI = ABIElementMarshaling{
```

Comment:
I note this might have a merge conflict with @nguyer who I believe is moving all of these to the [firefly-signer/pkg/api](https://pkg.go.dev/github.com/hyperledger/firefly-signer/pkg/abi) package.

... or if it doesn't then this is another place we should remove a duplicate interface definition.

Reply:
looks like this one is ready first - sorry @nguyer